src/metabase/db/custom_migrations.clj

```diff
@@ -105,6 +105,38 @@
   (when s
     (.toLowerCase (str s) Locale/US)))
 
+(defn json-in
```

Comment:
copied from metabase.models.interface

Reply:
Probably worth adding a reference to the corresponding methods in the docstrings.